src/test/unit/algorithms/CondProbTableTest.cpp

```diff
@@ -26,14 +26,28 @@
 
 #include <nupic/algorithms/CondProbTable.hpp>
 #include <nupic/math/StlIo.hpp>
-#include "CondProbTableTest.hpp"
 #include <fstream>
 #include <boost/numeric/ublas/storage.hpp>
-#include <boost/numeric/ublas/matrix_proxy.hpp>
+#include "gtest/gtest.h"
+
 
 using namespace std;
 using namespace boost;
+
+
 namespace nupic {
+  
+  class CondProbTable;
+
+  // // Size of the table we construct
+  Size numRows() {return 4;}
```

Comment:
@scottpurdy It's defined in https://github.com/numenta/nupic.core/blob/master/src/nupic/types/Types.hpp#L135